google/cloud/storage/internal/curl_client.cc

```diff
@@ -516,7 +527,7 @@ std::pair<Status, ObjectMetadata> CurlClient::CopyObject(
 std::pair<Status, ObjectMetadata> CurlClient::GetObjectMetadata(
     GetObjectMetadataRequest const& request) {
   CurlRequestBuilder builder(storage_endpoint_ + "/b/" + request.bucket_name() +
-                                 "/o/" + request.object_name(),
+                                 "/o/" + EscapeObjectName(request),
```

Comment:
I think keeping this `UrlEscapeString(request.object_name())` is more consistent than having special member functions.

Reply:
Done.